modules/custom/lua/dynamic_lottery_chance.lua

```diff
@@ -0,0 +1,232 @@
+-----------------------------------
+-- Dynamic lottery chance for placeholder-based NMs
+--
+-- 
+-----------------------------------
+require("modules/module_utils")
+-----------------------------------
+local m = Module:new("dynamic_lottery_chance")
+
+-- HOW TO USE
+--
+-- 1. Define a defaultRule (detailed below) applied to all NMs, or set defaultRule to nil if you want to specify custom behavior per-NM
```

Comment:
Should probably just remove the bloat then and only have one maintained table: 

Nms to include - Its very very not need to add to EVERY nm in the game blanket wise

Reply:
Bloat-wise it will really only remove about ~5 lines from the entire file (plus the default exclusion list I added). I think there's value in the default rule, as it's a general tool to reduce variance for all PH NMs while being able to keep average spawn time similar to their current values, and it's pretty easily bypassed by just leaving the default rule as nil.
But if that's the difference of using it or not, I can just rip it out. It would also be pretty easy for a server owner to add the functionality back in if they wanted to.

Reply:
Less bloat is better in this case